desc/compute/data_index.py

```diff
@@ -18,6 +26,12 @@
     "description": "Radial coordinate, proportional to the square root of the toroidal flux",
     "fun": "compute_flux_coords",
     "dim": 1,
+    "dependencies": {
+        "params": [],
+        "transforms": {"grid": []},
```

Comment:
what does having a "grid" in transforms dict do? does this specify that ONLY the grid is used from the transform and not any of the evaluated coefficients/derivaitves ?

Reply:
Yeah basically. There are some things that need to know about the grid, and we used to usually get the one from R transformer, but now we won't always have any given transform, so figured it's better to just rely on the grid by itself (which should be the same as the grid in all the transforms)